docs/cli.md

```diff
@@ -157,7 +157,7 @@ Sets the project name, which is prepended to the name of every container started
 
 ### COMPOSE\_FILE
 
-Sets the path to the `docker-compose.yml` to use. Defaults to `docker-compose.yml` in the current working directory.
+Sets the path to the `docker-compose.yml` to use. Defaults to `docker-compose.yml`, that will be searched in the current working directory and then in each parent directory.
```

Comment:
Wording's a bit awkward. Also, this should be wrapped to 80 chars. How about:

    Specify what file to read configuration from. If not provided, Compose will look
    for `docker-compose.yml` in the current working directory, and then each parent
    directory successively, until found.